src/test/resources/cucumber/features/integrationTests/ast/astScanProcessing.feature

```diff
@@ -1,35 +1,44 @@
 @AstIntegrationTests @IntegrationTest
 Feature: Cx-Flow AST Integration permutation tests
-  
 
-  @ASTRemoteRepoScan 
-  Scenario Outline: scan multiple initiators
-    Given scan initiator list is "<initiators>"
-    Then the returned contain populated results for all initiators
+
+  @ASTRemoteRepoScan
+  Scenario Outline: using multiple vulnerability scanners
+    Given enabled vulnerability scanners are "<scanners>"
+    Then scan results contain populated results for all scanners
     And sca finding count will be "<sca_findings>" and ast findings count "<ast_findings>" will be accordingly
 
     Examples:
-      | initiators | sca_findings | ast_findings |
-      | SCA        | 13           | 0            |
-      | AST        | 0            | 11           |
-      | AST,SCA    | 13           | 11           |
+    # Cannot rely on an exact number of findings, because it may change after backend version updates.
+      | scanners | sca_findings | ast_findings |
+      | SCA      | > 0          | 0            |
+      | AST      | 0            | > 0          |
+      | AST,SCA  | > 0          | > 0          |
 
-  @ASTRemoteRepoScan
-  Scenario Outline: check error message on expired AST token
-    Given AST scan with expired token
-    Then expired token expected error will be returned "<message>"
+  @ASTRemoteRepoScan @InvalidCredentials
+  Scenario Outline: Trying to scan with invalid credentials
+    When CxFlow tries to start AST scan with the "<client id>" and "<client secret>" credentials
```

Comment:
Given AST scan with the "<client id>" and "<client secret>" credentials
When running scan